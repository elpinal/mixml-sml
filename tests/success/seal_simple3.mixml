link
  link X1 = {module t = [type :Type]} with
  link X2 = {module u = [type int]} with
  {module f = [val :ext X2.u -> ext X1.t]}
seals

link {module t = [type bool]} with
link Y2 = {module u = [type :Type]} with
{module f = [val fun (x : ext Y2.u) -> true]}
