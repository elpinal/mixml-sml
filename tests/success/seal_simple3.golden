{f : [int -> !0]+, t : [= !0], u : [= int]}
